### PR TITLE
Add typings for vexdb

### PR DESCRIPTION
**Pull request by brenapp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,9 +0,0 @@
-root = true
```
Comment (plantain-00) [inline .editorconfig]: I think this file shouldn't be removed

File: types/vexdb/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Installation
```
Comment (plantain-00) [inline types/vexdb/README.md]: I think the `README.md` is not required here, there will be one generated when published to npm

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: .editorconfig [hunk 3]
```diff
@@ -6,4 +6,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 
 [{*.json,*.yml}]
-indent_style = space
+indent_style = space
```
Comment (plantain-00) [inline .editorconfig]: Still, this file is not expected to be changed here

File: types/vexdb/.editorconfig [hunk 4]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[{*.json,*.yml}]
+indent_style = space
```
Comment (plantain-00) [inline types/vexdb/.editorconfig]: This file is not required here

**Comment by MayorMonty** [comment]
@plantain-00 I think was able to fix the automatic file generation of my editor, could this be approved?

**Comment by plantain-00** [comment]
The `.editorconfig` is still changed in this PR, I suggest not to.

**Pull request merged**